qiime2/plugins.py

```diff
@@ -188,12 +188,12 @@ def init_format(self, name, factory, ext=None):
         return variable
 
     def import_from_format(self, name, semantic_type,
-                           fmt_variable, view_type=None):
-        variable = super().import_from_format(
-            name, semantic_type, fmt_variable, view_type=view_type)
+                           variable, view_type=None):
+        var = super().import_from_format(
+            name, semantic_type, variable, view_type=view_type)
 
         interface_name = variable.to_interface_name()
```

Comment:
This line needs to become `var` now, because it's the resulting artifact variable (same as in q2cli).